sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -109,26 +142,39 @@ Both the asynchronous and synchronous Service Bus sender and receiver clients ca
 
 The snippet below creates an asynchronous Service Bus sender.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L18-L23 -->
+<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L21-L26 -->
 ```java
-String connectionString = "<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>";
-ServiceBusSenderAsyncClient sender = new ServiceBusClientBuilder()
-    .connectionString(connectionString)
-    .buildSenderClientBuilder()
-    .entityName("<< QUEUE OR TOPIC NAME >>")
-    .buildAsyncClient();
+ */
+public void createAsynchronousServiceBusSender() {
+    String connectionString = "<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>";
+    ServiceBusSenderAsyncClient sender = new ServiceBusClientBuilder()
+        .connectionString(connectionString)
+        .buildSenderClientBuilder()
```

Comment:
The namespace connection string is not sufficient, they need to also set the entityPath.

Reply:
I am setting `.queueName("<< QUEUE NAME >>")`  in updated sampels.